skbuild/cmaker.py

```diff
@@ -103,24 +102,45 @@ def configure(self, clargs=(), generator_id=None):
         if not os.path.exists(CMAKE_INSTALL_DIR):
             os.makedirs(CMAKE_INSTALL_DIR)
 
-        if not os.path.exists(DISTUTILS_INSTALL_DIR):
-            os.makedirs(DISTUTILS_INSTALL_DIR)
+        if not os.path.exists(SETUPTOOLS_INSTALL_DIR):
+            os.makedirs(SETUPTOOLS_INSTALL_DIR)
 
         python_version = CMaker.get_python_version()
         python_include_dir = CMaker.get_python_include_dir(python_version)
         python_library = CMaker.get_python_library(python_version)
 
-        cmd = ['cmake', os.getcwd(), '-G', generator_id,
-               '-DCMAKE_INSTALL_PREFIX:PATH={0}'.format(
-                    os.path.join(os.getcwd(), CMAKE_INSTALL_DIR)),
-               '-DPYTHON_EXECUTABLE:FILEPATH=' + sys.executable,
-               '-DPYTHON_VERSION_STRING:STRING=' + sys.version.split(' ')[0],
-               '-DPYTHON_INCLUDE_DIR:PATH=' + python_include_dir,
-               '-DPYTHON_LIBRARY:FILEPATH=' + python_library,
-               '-DSKBUILD:BOOL=TRUE',
-               "-DCMAKE_MODULE_PATH:PATH={}".format(
-                   os.path.dirname(__file__) + '/resources/cmake')
-               ]
+        cwd = os.getcwd()
+        cmd = ['cmake', cwd, '-G', generator_id]
+        cmd.extend(
+            "=".join((cmake_variable, value)) for cmake_variable, value in (
```

Comment:
Different strokes, I guess.  I had hoped this would make the arguments that we're passing easier to read through.